src/js/visual/configurator-csr.js

```diff
@@ -1025,34 +1025,37 @@ ripe.ConfiguratorCsr.prototype._initDebug = function() {
             this.debugRefs.renderedInitials.group.add(this.debugRefs.renderedInitials.axis);
         }
 
-        // inits reference points curve
-        if (this.debugOptions.renderedInitials.line) {
-            const curve = new window.THREE.CatmullRomCurve3(
-                this.initialsRefs.renderedInitials.points,
-                false,
-                "centripetal"
-            );
-            const pointsNum = 50;
-            const linePoints = curve.getPoints(pointsNum);
-            this.debugRefs.renderedInitials.line = new window.THREE.Line(
-                new window.THREE.BufferGeometry().setFromPoints(linePoints),
-                new window.THREE.LineBasicMaterial({ color: 0x0000ff })
-            );
-
-            this.debugRefs.renderedInitials.group.add(this.debugRefs.renderedInitials.line);
-        }
+        // ensures it has the minimum number of points
+        if (this.initialsRefs.renderedInitials.points > 1) {

```

Comment:
**For the reviewers:**
The fix is very small, it was just adding this check, the rest of the changes is just the code now being inside this if brackets